src/Microsoft.Windows.ComputeVirtualization/Container.cs

```diff
@@ -104,27 +139,32 @@ public Process CreateProcess(ProcessStartInfo startInfo)
                 RestrictedToken = startInfo.RestrictedToken,
                 User = startInfo.User,
                 WorkingDirectory = startInfo.WorkingDirectory,
+                CreateStdInPipe = startInfo.RedirectStandardInput,
+                CreateStdOutPipe = startInfo.RedirectStandardOutput,
+                CreateStdErrPipe = startInfo.RedirectStandardError,
             };
 
-            HcsFunctions.CreateProcessWithStdHandlesInComputeSystem(_id, JsonHelper.ToJson(parameters), out pid, stdinAddr, stdoutAddr, stderrAddr);
+            IntPtr process = new IntPtr();
```

Comment:
process is first passed as `out`. You don't need to initialize its just wasteful.